test/test_RequestApi.py

```diff
@@ -44,3 +46,12 @@ def test_replace_variable_sends_invalid_dictionary_and_input_endpoint_return_url
     request_api = RequestApi()
     actual = request_api.replace_variables(endpoint, id_dictionary)
     assert expected == actual
+
+
+def test_read_file_path_and_return_new_file_path():
+    file_path = "trello/jsonfiles/file.json"
+    new_file_path = path.realpath(file_path)
+    expected = {"name": "limbert"}
+    json_reader = JsonFileReader()
+    actual = json_reader.read(file_path)
+    assert expected == actual
```

Comment:
what is the objective of this unit test?

Reply:
return the new file path in dictionary